rental.rb

```diff
@@ -0,0 +1,7 @@
+class Rental
+  attr_accessor :date
+
+  def initialize(date)
```

Comment:
- [ ] please the constructor of Rental should be modified so Book and Person are set in it to break the many-to-many relationship. see more [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)
- note that the rental will be adding itself to the @book and @person achieving the 'belongs-to' association`